src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,225 @@
+/* stylelint-disable no-descending-specificity */
+
+.section-title {
+    display: flex;
+    margin: 30px 0 10px 16px;
+}
+
+.section-title__title {
+    max-width: 75%;
+
+    > h2 {
+        margin: 0;
+    }
+}
+
+.section-title__subtitle {
+    color: @color-grey5;
+}
+
+.section-title__cta {
+    margin: 0 16px;
+    white-space: nowrap;
+
+    > a {
+        border-left: 2px solid @color-grey2;
+        display: inline-block;
+        padding-left: 16px;
+    }
+}
+
+.section-title__cta--no-text {
+    margin: 0;
+
+    > a {
+        border: 0;
+        padding: 0;
+    }
+
+    > a > span:not(.section-title__cta-icon) {
+        display: none;
+    }
+}
+
+.section-title__cta > a,
+.section-title__title > h2 > a {
+    color: @color-black;
+    text-decoration: none;
+
+    &:hover {
+        color: @color-black;
+        text-decoration: underline;
+    }
+
+    &:visited {
+        color: @color-black;
+    }
+}
+
+.section-title__cta-icon {
+    display: inline-block;
+    margin-left: 8px;
+
+    span& {
```

Comment:
I would remove these nested ones and actually put them outside. 
It might be more lines, but I do think it improves readability. 
so
```
.section-title__cta-icon {}

span.section-title__cta-icon {}

svg.section-title__cta-icon {}
```

Reply:
Yes, this is a pattern that makes it easier to understand the selectors. I agree.